vmicore/src/os/windows/KernelOffsets.h

```diff
@@ -0,0 +1,121 @@
+#ifndef VMICORE_KERNELOFFSETS_H
+#define VMICORE_KERNELOFFSETS_H
+
+#include "../../vmi/LibvmiInterface.h"
+#include <memory>
+
+namespace KernelStructOffsets
+{
+    using _control_area = struct _control_area
+    {
+        addr_t _mmsection_flags;
+        addr_t FilePointer;
+    } __attribute__((aligned(16))); // NOLINT(cppcoreguidelines-avoid-magic-numbers)
```

Comment:
nit: I would simply enclose this namespace in `// NOLINTBEGIN(cppcoreguidelines-avoid-magic-numbers) [...] // NOLINTEND(cppcoreguidelines-avoid-magic-numbers)` instead of writing `// NOLINT(cppcoreguidelines-avoid-magic-numbers)` multiple times.